wiki/osu!api/id.md

```diff
@@ -0,0 +1,11 @@
+# osu!api
+
+osu! memiliki API publik yang dapat digunakan oleh layanan pihak ketiga untuk memperluas dan mendukung platform osu!. Dulu API bersifat lebih tertutup, hanya dapat digunakan sesuai permintaan, tetapi sekarang siapa pun dapat menggunakan API selama mereka memiliki akun osu!.
```

Comment:
Better using this sentences 

> osu!api dapat digunakan oleh siapapun selama mereka memiliki akun osu!, berbeda dengan perizinan API pada sebelumnnya yang hanya dapat digunakan sesuai dengan permintaan kerena bersifat tertutup.

`... Dulu API bersifat lebih tertutup, ...` << Look like jumping

PS : The sentences wont like raw translate but more understandable